src/sinol_make/__init__.py

```diff
@@ -9,7 +9,7 @@
 from sinol_make import util, oiejq
 
 
-__version__ = "1.5.8"
+__version__ = "1.5.9"
```

Comment:
I think it's common to first merge all PRs and then create a tiny PR that just bumps the version. This approach might be useful in your case.

Reply:
Sure

Reply:
Sure